### DISCUSSION TITLE
Categories View: Column Sums

### DISCUSSION THREAD

**Discussion by dautrich** [joomla/joomla-cms#47033] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
No

### Describe the solution you'd like
##Categories View in Backend
The Categories View in backend shows the count of Published, Unpublished, Archived and Trashed articles in each category.
![grafik](https://github.com/user-attachments/assets/1cb34260-c79f-49b4-8401-80416bb7833c)

If there are numerous categories, it would be nice to get sums for each of the four columns mentioned. This might even be helpful in filtered views.

**Comment by brianteeman** [comment]
why - what is the use case for this. what happens on page 2, 3 etc

**Comment by dautrich** [comment]
>  what happens on page 2, 3 etc

The line with the sums could be displayed above the categories list.
